packages/fxa-settings/src/components/FormResetPasswordWithBalloon/index.tsx

```diff
@@ -54,7 +55,7 @@ export const FormResetPasswordWithBalloon = ({
   return (
     <>
       <form {...{ onSubmit }} className="flex flex-col gap-4">
-        <div className="tooltip-container">
+        <div className="relative">
```

Comment:
Man, it'd be nice if this class could be used in the component that's using it (wrapping `PasswordStrengthBalloon` instead) so someone doesn't come across this and accidentally remove it since they don't see its purpose, but I think you'd have to recalculate a bunch of values. Maybe a comment here would be good even if it's just something like, "for PasswordStrengthBalloon positioning" 🤔 

One other thing that could help would be having `PasswordStrengthBalloon` in Storybook under components. You could wrap it in a very basic input with that `relative` class showing too so if someone's like, "how do I use this thing?" they can check out the component in Storybook and see it at its basic usage.

Reply:
I agree the separation of concerns isn't great here, but it seemed like the simplest solution to implement.

I do have a storybook component for the PasswordStrengthBalloon with a few example password values:
![image](https://user-images.githubusercontent.com/22231637/213053320-77b06392-6f2c-455f-95b4-f28fa3241c4a.png)

I'll definitely add comments about usage in the FormResetPasswordWithBalloon component and the PasswordStrengthBalloon story.